### PR TITLE
Tidy companies transformers

### PR DESCRIPTION
**Pull request by tomkingston**
## Description of change

Refactor: tidies the new react companies list page's transformers and state management.

## Test instructions

The react companies list page should show a list of paginated results (note the filters have not been added yet), url: /companies/react

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to master?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/apps/companies/client/transformers.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-const labels = require('../labels')
-const urls = require('../../../lib/urls')
+import labels from '../labels'
+import urls from '../../../lib/urls'
 
-const { addressToString } = require('../../../common/addresses')
-const { formatLongDate, formatMediumDateTime } = require('../../../common/date')
+import { addressToString } from '../../../common/addresses'
+import { formatLongDate, formatMediumDateTime } from '../../../common/date'
```
Comment (paulgain) [inline src/apps/companies/client/transformers.js:7]: Both of these shouldn't be referencing `/src/common/*` as that's specifically for the node layer, you should be referencing dates from `src/client/utils`.

File: src/apps/companies/client/state.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ export const TASK_GET_COMPANIES_LIST = 'TASK_GET_COMPANIES_LIST'
 
 export const ID = 'companiesList'
 
-const searchParamProps = ({ page = 1 }) => ({ page })
+const searchParamProps = ({ page = 1 }) => ({ page: parseInt(page) })
```
Comment (paulgain) [inline src/apps/companies/client/state.js:7]: 👍

File: src/apps/companies/client/tasks.js [hunk 3]
```diff
@@ -2,6 +2,8 @@ import axios from 'axios'
 
 const handleError = (error) => Promise.reject(Error(error.response.data.detail))
 
+import { transformResponseToCompanyCollection } from './transformers'
```
Comment (paulgain) [inline src/apps/companies/client/tasks.js:5]: Imports should sit above the constants.

Comment (tomkingston) [reply src/apps/companies/client/tasks.js:5]: Oops!

**Review by paulgain — APPROVED** [review]
LGTM

**Review by ian-leggett — APPROVED** [review]

**Pull request merged**